src/libaktualizr/primary/aktualizr.h

```diff
@@ -159,6 +159,13 @@ class Aktualizr {
    */
   void UptaneCycle();
 
+  // reboot handling
+
+  bool RebootDetected();
+
+  void RebootFlagSet();
+  void RebootFlagClear();
+
```

Comment:
What are these for? Do they need to be part of the API?

Reply:
Remnants of a first design, they should go away (for now at least). Thanks!